src/core/SliceRepresentation.js

```diff
@@ -111,8 +115,6 @@ export default class SliceRepresentation extends Component {
       changed = true;
       const preset = vtkColorMaps.getPresetByName(colorMapPreset);
       this.lookupTable.applyColorMap(preset);
-      this.lookupTable.setMappingRange(...colorDataRange);
-      this.lookupTable.updateRange();
     }
```

Comment:
Should you keep the scaling on that colorDataRange?

Reply:
Nope - That happens here: https://github.com/Kitware/react-vtk-js/blob/e5d7e5f6fa934288c8f53adc3567bdf8a79d04a9/src/core/SliceRepresentation.js#L126-L146